props/configs/src/main/resources/calendar/UTC.calendar

```diff
@@ -1,11 +1,14 @@
 <!--
-  ~ Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+  ~ Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
   -->
 
 <calendar>
     <name>UTC</name>
     <timeZone>UTC</timeZone>
+    <description>UTC</description>
+    <firstValidDate>1500-01-01</firstValidDate>
+    <lastValidDate>2500-12-31</lastValidDate>
     <default>
-        <businessPeriod>00:00,24:00</businessPeriod>
```

Comment:
Was this change necessitated from a parsing issue? IMO, the exact close time should be excluded (in which case, 24:00 is correct).

Reply:
The exact problem is a bit complex, and I keep waffling about the right solution.  I'm not yet convinced that the existing solution isn't right.  This is related to the open vs closed interval question.

Consider the case of the NYSE closing auction.  These trades occur and post at exactly 4PM.  If the interval is open, these trades are considered to be after business time.  If the interval is closed, they occur during business time.  I'm inclined to think that the closed interval is the correct interpretation, but I'm discussing this with @pete-petey and @rbasralian.  

Additionally:
1) The max value of `LocalTime` is `23:59:59.999999999`.  https://docs.oracle.com/javase/8/docs/api/java/time/LocalTime.html
2) Our `DateTimeUtils.parseLocalTime` is permissive and parses `24:00` as `00:00`.  Since it is a local time, the fact that `24:00` intends to be the next day is lost.
3) Because of the above, `00:00,24:00` was being parsed as a standard business day of `00:00,00:00`, which is clearly not what is desired.

It probably makes sense to reject `24:00` in the XML parser if the closed interval is the winner.  It may also make sense to reject `24:00` in  `DateTimeUtils.parseLocalTime`.

Reply:
Both @pete-petey and @rbasralian think the interval should be closed.  Changes have been made to make it easier to support open intervals in the future if anyone cares.  Also a proper error is raised on `24:00` since it is on the next day.